oracle/oracle-how-to/use-bastion-to-access-VM.rst

```diff
@@ -1,25 +1,34 @@
 Use a bastion to access your VM
-===============================
+=============================== 
 
 To increase security and avoid Internet based attacks, it might make sense to run your VMs without a public IP address and just use a private IP instead. The only way to access such a VM will be through your private network in the Oracle Cloud VNC (Virtual Cloud Network).
 
-`Oracle's bastion feature`_ provides a time-limited access to VMs without a public endpoint. Bastions control authorised users and allow them to SSH into a VM from specific IP addresses. It also negates the need to maintain an instance with a public and private address to act as a bastion (which in turn runs the risk of Internet exposure).
+`Oracle's bastion feature`_ provides a time-limited access to VMs that do not have a public endpoint. Bastions control authorised users and allow them to SSH into a VM from specific IP addresses. It also negates the need to maintain an instance with a public and private address to act as a bastion (which in turn runs the risk of Internet exposure).
 
-To use a bastion, as a prerequisite you'll need your VCN to include a gateway (a service gateway, an internet gateway, or a NAT gateway) and a route rule for the gateway. For details on how to handle these services, refer to the Oracle Cloud documentation at `Service Gateway`_, `Internet Gateway`_, or `NAT Gateway`_.
+Prerequisites
+-------------
+
+To use a bastion, you'll need a gateway in your VCN and a route rule for the gateway in your VM. The gateway could be a service gateway, an internet gateway, or a NAT gateway. The setup will work in any of the following combinations:
```

Comment:
Should VCN be defined? I'm unclear on what it means. Virtual Cloud Network? Would VPC would be a more industry-standard term?

Reply:
The full form, 'Virtual Cloud Network' is mentioned in the introductory paragraph and it seems to be the standard terminology used in Oracle Cloud (instead of VPC). Since this how-to guide is mainly intended for users of Oracle Cloud, it makes sense to use their terminology. WDYT?

(Also, the introductory para had a typo and it mentioned VNC instead of VCN. It is now corrected.)